src/connector/src/parser/protobuf/parser.rs

```diff
@@ -1137,13 +926,13 @@ mod test {
                     assert_eq!(
                         jv,
                         JsonbVal::from(json!({
-                            "_type": "test.AnyValue",
-                            "any_value_1": {
-                                "_type": "test.StringValue",
+                            "@type": "type.googleapis.com/test.AnyValue",
+                            "anyValue1": {
+                                "@type": "type.googleapis.com/test.StringValue",
                                 "value": "114514",
                             },
-                            "any_value_2": {
-                                "_type": "test.Int32Value",
+                            "anyValue2": {
+                                "@type": "type.googleapis.com/test.Int32Value",
```

Comment:
> Is it possible to put a simple example to show what's the difference between the previous JSON value and the current JSON value in this PR.

This unit test can serve as an example of the differences. As mentioned in PR description, I also plan to reorganize the tests in a follow-up to make it easier to focus on test input & output rather than repeated test construction code.

And possibly more tests to demonstrate how it handles the special case https://github.com/risingwavelabs/risingwave/pull/12291#discussion_r1378007787 as well as the `potential naming conflict` mentioned in previous `TODO`.

Reply:
Thank you so much for your explanations!